addon/validators/base.js

```diff
@@ -128,14 +121,19 @@ const Base = Ember.Object.extend({
           return OptionsClass.create();
         }
 
-        return Ember.Object.create(Object.keys(builtOptions).reduce((obj, o) => {
+        var opts = Ember.Object.create();
+        var appliedOpts = Object.keys(builtOptions).reduce((obj, o) => {
           obj[o] = get(this, o);
           return obj;
-        }, {}));
+        }, {});
+        opts.setProperties(appliedOpts);
+        return opts;
       }
     });
 
-    return Object.freeze(OptionsClass.create());
+    var optionsInstance = OptionsClass.create();
+    optionsInstance.setProperties(builtOptions);
+    return optionsInstance;
   },
```

Comment:
@xcambar please revert all changes made to `base.js`. Everything I did in there was well thought out and intentional. If you would like, I can give you a detailed explanation as to why I'm doing all of this. If you're on ember slack, we can just chat on there 😸